rest/data/commit_queue.go

```diff
@@ -397,6 +397,13 @@ func getPRAndCheckMergeable(ctx context.Context, env evergreen.Environment, sc C
 				"pr":      info.PR,
 			}))
 		} else if p != nil && p.Version != "" {
+			grip.Debug(message.Fields{
+				"message": "calling refresh from commit queue",
```

Comment:
might also be interesting to log if this actually changes this status, so maybe logging after 412 instead and logging mergeableState

Reply:
moved and printing new mergable state!